src/main/java/com/duckblade/osrs/toa/module/TombsOfAmascutModule.java

```diff
@@ -78,6 +79,7 @@ protected void configure()
 		lifecycleComponents.addBinding().to(TargetTimeManager.class);
 		lifecycleComponents.addBinding().to(UpdateNotifier.class);
 		lifecycleComponents.addBinding().to(SarcophagusRecolorer.class);
+		lifecycleComponents.addBinding().to(CursedPhalanxDetector.class);
```

Comment:
nitpick: I prefer alphabetical order for this list, but I realize SarcophagusRecolorer is already wrong as well